src/feedparser_clj/core.clj

```diff
@@ -94,13 +94,20 @@
         syndfeed (.build feedinput xmlreader)]
     (make-feed syndfeed)))
 
+(defn string->url [s]
+  (if (string? s)
+    (URL. s)
+    s))
+
 (defn parse-feed "Get and parse a feed from a URL"
   ([feedsource]
-     (parse-internal (new XmlReader (if (string? feedsource)
-                                      (URL. feedsource)
-                                      feedsource))))
+   (parse-internal (XmlReader. (string->url feedsource))))
   ([feedsource content-type]
-     (parse-internal (new XmlReader feedsource content-type))))
+   (parse-internal (XmlReader. (string->url feedsource) content-type)))
+  ([deefsource content-type lenient]
```

Comment:
@Gonzih you have a typo here and below. Should be `feedsource` instead of `deefsource`.